node/staking.go

```diff
@@ -29,6 +29,7 @@ const (
 // UpdateStakingList updates staking information by querying the staking smart contract.
 func (node *Node) UpdateStakingList(stakeInfoReturnValue *structs.StakeInfoReturnValue) {
 	node.CurrentStakes = make(map[common.Address]*structs.StakeInfo)
+	node.CurrentStakesByNode = make(map[[20]byte]*structs.StakeInfo)
```

Comment:
why hardcode the [20]byte ?

Reply:
It's a convention copied from elsewhere where we store BLS addresses.  Proper fix requires replacing all occurrences of `[20]byte` for BLS addresses throughout the code.  Will track this in #670.